style.css

```diff
@@ -0,0 +1,404 @@
+  font-size: 1.7rem;
+  cursor: pointer;
+  font-weight: 500;
+}
+
+.btn:hover {
+  background: var(--pink);
+}
+
+.header .header-1 {
+  background: #fff;
+  padding: 1.5rem 9%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.header .header-1 .logo {
+  font-size: 2.5rem;
+  font-weight: bolder;
+  color: var(--default-color);
+}
+
+.header .header-1 .logo i {
+  color: var(--pink);
+}
+
+.header .header-1 .search-form {
+  width: 50rem;
+  height: 5rem;
+  border: var(--border);
+  overflow: hidden;
+  background: #fff;
+  display: flex;
+  align-items: center;
+  border-radius: 0.5rem;
+}
+
+.header .header-1 .search-form input {
+  font-size: 1.6rem;
+  padding: 0 1.2rem;
+  height: 100%;
+  width: 100%;
+  text-transform: none;
+  color: var(--black);
+}
+
+.header .header-1 .search-form label {
+  font-size: 2.5rem;
+  padding-right: 1.5rem;
+  color: var(--black);
+  cursor: pointer;
+}
+
+.header .header-1 .search-form label:hover {
+  color: var(--pink);
+}
+
+.header .header-1 .icons div,
+.header .header-1 .icons a {
+  font-size: 2.5rem;
+  margin-left: 1.5rem;
+  color: var(--black);
+  cursor: pointer;
+}
+
+.header .header-1 .icons div:hover,
+.header .header-1 .icons a:hover {
+  color: var(--pink);
+}
+
+#search-btn {
+  display: none;
+}
+
+.header {
+  background: var(--pink);
+}
+
+.header .navbar {
+  text-align: center;
+}
+
+.header .navbar a {
+  color: #fff;
+  display: inline-block;
+  padding: 1.2rem;
+  font-size: 1.7rem;
+}
+
+.header .navbar a:hover {
+  background: var(--dark-color);
+}
+
+.header {
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  z-index: 1000;
+}
+
+.arrival {
+  margin-top: 5rem;
+}
+.arrival .box-container {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-wrap: wrap;
+  padding: 2rem 0;
+  width: 85%;
+  margin: 2rem auto;
+}
+
+.arrival .box-container .box {
+  flex: 1 1 20rem;
+  border: 0.1rem solid rgba(0, 0, 0, 0.3);
+  overflow: hidden;
+  position: relative;
+  margin: 1rem 2rem;
+}
+
+.arrival .box-container .box .image {
+  height: auto;
+  padding: 2rem;
+  text-align: center;
+  background: #fff;
+}
+
+.arrival .box-container .box .image img {
+  max-width: 100%;
+  height: auto;
+}
+
+.arrival .small-txt {
+  background-color: var(--bg-color);
+  color: var(--font-color);
+  font-size: 1rem;
+  padding: 5px 10px;
+  position: absolute;
+  left: 0;
+  bottom: 4.3rem;
+  width: 100%;
+  text-align: center;
+}
+
+.feature .row {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-wrap: wrap;
+  padding: 2rem 0;
+  margin: 1rem auto;
+  width: 95%;
+}
+
+.feature .image-container {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-flow: column wrap;
+  padding: 2rem 0;
+  margin: 1rem auto;
+}
+
+.feature .banner-img {
+  max-width: 100%;
+  height: auto;
+  border: 0.1rem solid rgba(0, 0, 0, 0.3);
+  box-shadow: 0px 4px 50px rgba(0, 0, 0, 0.1);
```

Comment:
use color variable or if you are using this color only here than give the comment

Reply:
done